openlibrary/templates/type/author/view.html

```diff
@@ -144,18 +144,9 @@ <h2 class="collapse">
                     $ungettext("1 work", "%(count)d works", books.num_found, count=books.num_found)
                     <span class="count smaller"><a href="/books/add?author=$page.key">$_('Add another?')</a></span>
                 </h2>
-                <p id="books"><span class="tools">
+                <p id="books">
                 $if books.num_found > 1:
-                    <img src="/images/icons/icon_sort.png" alt="$_('Sorting by')" width="9" height="11" style="margin-right:5px;"/>
-                    $if books.sort == 'editions':
-                        <strong class="lightgreen">$_('Most Editions')</strong> | <a href="$changequery(sort='old')#editions">$_('First Published')</a> | <a href="$changequery(sort='new')#editions">$_('Most Recent')</a>
-                    $elif books.sort == 'old':
-                        <a href="$changequery(sort=None)#editions">$_('Most Editions')</a> | <strong class="lightgreen">$_('First Published')</strong> | <a href="$changequery(sort='new')#editions">$_('Most Recent')</a>
-                    $elif books.sort == 'new':
-                        <a href="$changequery(sort=None)#editions">$_('Most Editions')</a> |  <a href="$changequery(sort='old')#editions">$_('First Published')</a> | <strong class="lightgreen">$_('Most Recent')</strong>
-                    $elif books.sort == 'title':
-                        <a href="$changequery(sort=None)#editions">$_('Most Editions')</a> |  <a href="$changequery(sort='old')#editions">$_('First Published')</a> | <a href="$changequery(sort='new')#editions">$_('Most Recent')</a> | <strong class="lightgreen">$_('Title')</strong>
-                </span>
+                    $:render_template("search/sort_options.html", exclude='relevance', default_sort='editions')
```

Comment:
@Yashs911 You forgot to implement the `selected_sort` parameter in the template. This should resolve your issue.

Reply:
@lephemere No luck.
![image](https://user-images.githubusercontent.com/64412143/113482784-d9477200-94bd-11eb-96ca-4fbe67d6b268.png)

Reply:
Since `selected_sort` is not optional, you have specify explicitly, so:

```suggestion
                    $:render_template("search/sort_options.html", books.sort, exclude='relevance', default_sort='editions')
```

(Sort of like if you `sort_options.html` was a python function.